lib/additionals.bash

```diff
@@ -0,0 +1,16 @@
+_bash-it-additional-reload() {
```

Comment:
Why put this into a new file? Can't this go into the existing `helpers.bash` file? In this case, the `additional` part could be dropped as well.